{
  "_from": "gulp-series",
  "_id": "gulp-series@1.0.2",
  "_inBundle": false,
  "_integrity": "sha1-gWGZA1AXh13QDUiIklBP659jCgs=",
  "_location": "/gulp-series",
  "_phantomChildren": {},
  "_requested": {
    "type": "tag",
    "registry": true,
    "raw": "gulp-series",
    "name": "gulp-series",
    "escapedName": "gulp-series",
    "rawSpec": "",
    "saveSpec": null,
    "fetchSpec": "latest"
  },
  "_requiredBy": [
    "#DEV:/",
    "#USER"
  ],
  "_resolved": "https://registry.npmjs.org/gulp-series/-/gulp-series-1.0.2.tgz",
  "_shasum": "816199035017875dd00d488892504feb9f630a0b",
  "_spec": "gulp-series",
  "_where": "E:\\GitHub\\edmond-wu.github.io",
  "author": {
    "name": "Eugene Jo",
    "email": "iameugenejo@gmail.com",
    "url": "https://github.com/iameugenejo"
  },
  "bugs": {
    "url": "https://github.com/iameugenejo/gulps/issues"
  },
  "bundleDependencies": false,
  "deprecated": false,
  "description": "Gulp non-dependent ordered tasks runner. It solves the issue with the run-sequence module that doesn't abort the task chain on error.",
  "devDependencies": {
    "chai": "^1.9.1",
    "mocha": "^1.21.4"
  },
  "homepage": "https://github.com/iameugenejo/gulps",
  "keywords": [
    "gulp",
    "series",
    "sequence",
    "run-sequence"
  ],
  "license": "MIT",
  "main": "lib/gulp-series.js",
  "name": "gulp-series",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/iameugenejo/gulps.git"
  },
  "scripts": {
    "test": "mocha"
  },
  "version": "1.0.2"
}
